docs/modules/ROOT/examples/defaults_test.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+var (
+	testPath = "../../compiled/espejo/espejo"
+)
+
+func Test_Deployment_DefaultParameters(t *testing.T) {
+
+	subject := DecodeDeployment(t, testPath+"/10_deployment.yaml")
+	container := subject.Spec.Template.Spec.Containers[0]
```

Comment:
So this should be enough:
```suggestion
        require.NotEmpty(t, subject.Spec.Template.Spec.Containers)
	container := subject.Spec.Template.Spec.Containers[0]
```